samcli/lib/utils/async_utils.py

```diff
@@ -1,15 +1,18 @@
 """
 Contains asyncio related methods and helpers
 """
-import asyncio
 import logging
+from asyncio import AbstractEventLoop, gather, new_event_loop
 from concurrent.futures.thread import ThreadPoolExecutor
 from functools import partial
+from typing import Callable, List, Optional
 
 LOG = logging.getLogger(__name__)
 
 
-async def _run_given_tasks_async(tasks, event_loop=asyncio.get_event_loop(), executor=None):
+async def _run_given_tasks_async(
+    tasks: List[Callable], event_loop: AbstractEventLoop, executor: Optional[ThreadPoolExecutor] = None
```

Comment:
Did we use the default value at all?

Reply:
No, since the creation of this util the event loop was always specified: https://github.com/aws/aws-sam-cli/blob/bbaefffe3dd3e15741db5fbf422255df0ad4bd00/samcli/lib/utils/async_utils.py#L127-L131